buildh/UI.py

```diff
@@ -68,9 +77,14 @@ def parse_cli():
     parser.add_argument("-t", "--traj", type=isfile,
                         help="Input trajectory file. Could be in XTC, TRR or DCD format.")
     parser.add_argument("-l", "--lipid", type=str, required=True,
-                        help="Residue name of lipid to calculate the OP on (e.g. POPC).")
+                        help="Combinaison of the name of the ForceField "
+                        "and the residue name of lipid to calculate the OP on (e.g. Berger_POPC)."
+                        "It must match with the internal topology files or the one(s) supplied."
+                        "A list of supported terms is printed when calling the help.")
     parser.add_argument("-lt", "--lipid_topology", type=isfile, nargs='+',
-                        help="User topology lipid json file(s). Mandatory to build hydrogens.")
+                        help="User topology lipid json file(s). "
+                        "Some topology are already present internally."
+                        "Mandatory to build hydrogens.")
```

Comment:
I would remove "Mandatory to build hydrogens", I find it confusing since it's not a mandatory option! I'd rather say it's optional.

Reply:
Yeah, I found confusing too. I'll remove.